scipy/optimize/_minimize.py

```diff
@@ -798,6 +808,10 @@ def standardize_bounds(bounds, x0, meth):
         if not isinstance(bounds, Bounds):
             lb, ub = old_bound_to_new(bounds)
             bounds = Bounds(lb, ub)
+    elif meth == 'powell':
```

Comment:
why the double conversion here? With this PR `powell` will require a `Bounds` instance. So should this section just be:
```
    if meth in ['trust-constr', 'powell']:
        if not isinstance(bounds, Bounds):
            lb, ub = old_bound_to_new(bounds)
            bounds = Bounds(lb, ub)
```

Reply:
Fixed, thanks for pointing that out.